### DISCUSSION TITLE
docs about depNameSanitized

### DISCUSSION THREAD

**Discussion by satanTime** [renovatebot/renovate#7995] [Request Help] [answered]
Hi guys,

I want to group all `@angular/` updates into a single PR (per baseDir), for some reason the are separate ones.
that's my config.
```json
{
  "rangeStrategy": "pin",
  "rebaseWhen": "conflicted",
  "separateMajorMinor": true,
  "separateMinorPatch": true,
  "separateMultipleMajor": true,
  "includeForks": true,
  "additionalBranchPrefix": "{{#if baseDir}}{{baseDir}}/{{else}}root/{{/if}}"
}
```

Is there a place with explanation how `depNameSanitized` might be changed?

Thank you in advance

**Comment by rarkins** [comment, marked as answer]
You cannot change `depNameSanitized`, and that is not the way you want to do it anyway. Put them into a group instead. Add [group:angularMonorepo](https://docs.renovatebot.com/presets-group/#groupangularmonorepo) to your `extends` array or instead just extend `config:base` and get it and other monorepo definitions included.

**Reply by satanTime** [reply]
Thanks! Isn't `config:base` default behavior if I don't specify `extends` param in my config?

**Reply by rarkins** [reply]
No, you need to include it - it's opinionated and differs from the bot's default values

**Reply by satanTime** [reply]
Ah, good, thanks for the info, testing it currently.

A confusing thing was that with an empty config this grouping feature worked. But once I provided some unrelated values - it stopped.
I'll close the answer once I've got the desired behavior.

Thanks again and have a good day!

**Reply by rarkins** [reply]
Renovate's default behavior is to group nothing, so if there was grouping then it was definitely due to config you added which is not in the config you included above.

**Reply by satanTime** [reply]
Yep, you are right, the default config has this preset and I removed it later. Now it works as expected. Thanks!
